modules/RsIskraMeter/src/main.rs

```diff
@@ -893,6 +1041,223 @@ impl generated::OnReadySubscriber for IskraMeter {
 
 impl generated::SerialCommunicationHubClientSubscriber for IskraMeter {}
 
+impl generated::DisplayMessageServiceSubscriber for IskraMeter {
+    fn clear_display_message(
+        &self,
+        _context: &generated::Context,
+        _request: generated::types::display_message::ClearDisplayMessageRequest,
+    ) -> everestrs::Result<generated::types::display_message::ClearDisplayMessageResponse> {
+        let lock = self
+            .state_machine
+            .lock()
+            .map_err(|_| ::everestrs::Error::InvalidArgument("Internal error"))?;
+
+        let StateMachine::ReadyState(ready_state) = &*lock else {
+            return Err(::everestrs::Error::InvalidArgument("Not initialized"));
+        };
+
+        // Clear the LCD display by clearing bit 3 in parameters register
+        let res = ready_state.clear_lcd_custom_string();
+
+        match res {
+            Ok(()) => Ok(
+                generated::types::display_message::ClearDisplayMessageResponse {
+                    status: generated::types::display_message::ClearMessageResponseEnum::Accepted,
+                    status_info: None,
+                },
+            ),
+            Err(e) => {
+                log::error!("Failed to clear display message {:?}", e);
+                Ok(
+                    generated::types::display_message::ClearDisplayMessageResponse {
+                        status:
+                            generated::types::display_message::ClearMessageResponseEnum::Unknown,
+                        status_info: None,
+                    },
+                )
+            }
+        }
+    }
+
+    fn get_display_messages(
+        &self,
+        _context: &generated::Context,
+        _request: generated::types::display_message::GetDisplayMessageRequest,
+    ) -> everestrs::Result<generated::types::display_message::GetDisplayMessageResponse> {
```

Comment:
add `use ` statements for these long types at the top to make them more readable.